File: sources/com/s8/web/front/carbide/cube/CubeElement.java
package com.s8.web.front.carbide.cube;

import com.s8.io.bohr.neon.core.NeBranch;
import com.s8.io.bohr.neon.core.NeObject;

public class CubeElement extends NeObject {

	
	public CubeElement(NeBranch branch, String typeName) {
		super(branch, typeName);
	}

	
	public void setZIndex(int zIndex) {
		vertex.setUInt8("zIndex", zIndex);
	}


}

File: web-sources/S8-pkgs-ui-carbide/asw/CubeElement.js

import { S8 } from '/S8-api/S8Context.js';
import { NeObject } from '/S8-core-bohr-neon/NeObject.js';



/**
 * 
 */
S8.page.CSS_import('/S8-pkgs-ui-carbide/cube/Cube.css');



export class CubeElement extends NeObject {



    constructor(){
        super();
    }

    S8_render(){ /* continuous rendering approach... */ }

    S8_set_zIndex(level){
        this.getEnvelope().style.zIndex = level;
    }

    S8_dispose(){ /* continuous rendering approach... */ }
}